docs/specs/0018-unify-delete-confirmation-and-screen-naming.md

```
@@ -0,0 +1,77 @@
+# SPEC: refactor(ui): unify delete confirmation flow and align screen naming
+
+## Problem
```

markdownlint-cli2 (0.23.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)